src/GUI.cpp

```diff
@@ -28,6 +34,200 @@ static constexpr EnumStringArray<Position, static_cast<size_t>(Position::WindowR
 static constexpr EnumStringArray<CornerPosition, static_cast<size_t>(CornerPosition::BottomRight) + 1> CORNER_POSITION_ITEMS{
 	"top-left", "top-right", "bottom-left", "bottom-right"};
 
+/* Table with mapping between profession and elite specialization pair and their abbreviation and icon resource ID.
+*  The resource is loaded at runtime, with the table updated with IconLoader texture unique IDs and a pointer to the texture.
+*  Until then, the texture pointer will be nullptr.
+*/
+static std::map<std::pair<Prof, SpecializationId>, SpecializationData> ProfessionEliteMapping{
```

Comment:
Is the assumption that this is only accessed during a frame (where we are single threaded) and during startup (where GUI callback can't be called, since it hasn't even been given out)? Add a comment please if so

Reply:
Done